prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="/prep-html-links/index.html"><div>Click here to go to the home page</div></a>
```

Comment:
Here you'll want to use the `..` syntax for making a relative URL in your `href` attribute.

```html
href="../index.html"
```

Otherwise, this link won't work properly unless Live Server is used.

Reply:
Thank You Tim ! 
I fixed my mistake and tested it out without live server :) 

I just instead making a specific commit like "Delete commented out code" i put the same comment as i did previously "Completed prep-html-links" sorry wont happen next time !

Looking for forward to tomorrow 
Have a Goodnight !